src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -367,12 +367,26 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
             elif threshold_type == ReleaseThresholdType.CRASH_FREE_SESSION_RATE:
                 metrics.incr("release.threshold_health_status.check.crash_free_session_rate")
                 query_window = query_windows_by_type[threshold_type]
-                sessions_data = fetch_sessions_data(
-                    end=query_window["end"],
-                    request=request,
-                    organization=organization,
-                    params=filter_params,
-                    start=query_window["start"],
+                try:
+                    sessions_data = fetch_sessions_data(
+                        end=query_window["end"],
+                        request=request,
+                        organization=organization,
+                        params=filter_params,
+                        start=query_window["start"],
+                    )
+                except Exception as exc:
+                    logger.exception(str(exc))
```

Comment:
i think this will raise a sentry error. is this API hit in the deploy pipeline?

Reply:
Looks like logger.exception is fine and doesn't raise any exceptions itself!

Reply:
ok great!